tests/bsim/bluetooth/host/misc/conn_stress/scripts/conn_stress.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+simulation_id="conn_stress"
+
+test_path="tests_bsim_bluetooth_host_misc_conn_stress"
+bsim_central_exe_name="bs_${BOARD_TS}_${test_path}_central_prj_conf"
+bsim_peripheral_exe_name="bs_${BOARD_TS}_${test_path}_peripheral_prj_conf"
+bsim_args="-RealEncryption=1 -v=5 -s=${simulation_id}"
```

Comment:
```suggestion
bsim_args="-RealEncryption=1 -v=2 -s=${simulation_id}"
```
Any reason why you set devices verbosity to 5?